CHANGELOG.md

```diff
@@ -3,13 +3,43 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.1
+
+_Sep 29, 2021_
+
+This version includes bugfixes and features already released in v5.x.
```

Comment:
Would it help to frame the message as, v5 is active, and we have cherry-picked what we could from it to the legacy v4 version?

```suggestion
This version backports bug fixes and features from the v5.x active release line.
```

Reply:
Should we make clear that might not have other v4 releases due to the complexity of cherry-picking?

```suggestion
This version backports bug fixes and some of the features from the v5.x active release line.
As the development of v5 evolves, newer fixes might not be cherry-picked to the legacy version.
To have access to the latest features it is encouraged to upgrade to MUI X v5 and MUI Core v5.
```

---

I created a [CodeSandbox](https://codesandbox.io/s/datagriddemo-material-demo-forked-drz2n?file=/src/demo.js) using MUI Core v4 and v5 and it seems to work fine. The only drawback is having to install both versions, but the benefit of getting new features pays off, I think. We could add a section to the docs explaining how to configure the prefixes.

Reply:
Yeah, agree, a second about how to use MUI Core v4 would help. I have simplified/improved the codesandbox: https://codesandbox.io/s/datagriddemo-material-demo-forked-8myv0?file=/src/demo.js.

- No more emotion cache provider
- Use createGenerateClassName from the v4 package.